server/jdkhttp-server/src/main/scala/sttp/tapir/server/jdkhttp/internal/JdkHttpToResponseBody.scala

```diff
@@ -49,7 +52,56 @@ private[jdkhttp] class JdkHttpToResponseBody extends ToResponseBody[JdkHttpRespo
           .getOrElse {
             (base, Some(tapirFile.file.length()))
           }
-      case _: RawBodyType.MultipartBody => throw new UnsupportedOperationException("MultipartBody is not supported")
+      case m: RawBodyType.MultipartBody =>
+        val entity = MultipartEntityBuilder.create()
+        v.flatMap(rawPartToFormBodyPart(m, _)).foreach { (formBodyPart: FormBodyPart) => entity.addPart(formBodyPart) }
+        val builtEntity = entity.build()
+        val inputStream: InputStream = builtEntity.getContent
```

Comment:
Ah now I see that you're useing the httpcomponents lib for much more than plain mime-type mapping. That's fine then, let's keep it.